app/frontend/src/features/messages/surfing/HostRequestView.tsx

```diff
@@ -37,8 +37,10 @@ export default function HostRequestView({ hostRequest }: HostRequestViewProps) {
   }, [hostRequest.hostRequestId]);
 
   useEffect(() => {
-    dispatch(fetchUsers({ userIds: [hostRequest.fromUserId, hostRequest.toUserId] }));
-  }, [hostRequest.fromUserId, hostRequest.toUserId]);
+    dispatch(
+      fetchUsers({ userIds: [hostRequest.fromUserId, hostRequest.toUserId] })
+    );
+  }, [dispatch, hostRequest.fromUserId, hostRequest.toUserId]);
```

Comment:
I'm not too fussed but some other devs prefer unrelated things to be in a different PR. (Or maybe a seperate commit)

Reply:
Good point, it does make PRs more difficult to review otherwise if this is larger. I guess since this is a small addition that doesn't change much than pleasing the linter... I'm going to leave it in for this time but I'll try not do that in future!